src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/AbstractJavadocCheckTest.java

```diff
@@ -269,6 +272,33 @@ public void testVisitLeaveToken()
                 JavadocVisitLeaveCheck.visitCount, JavadocVisitLeaveCheck.leaveCount);
     }
 
+    @Test
+    public void testUnclosedTagIntolerantCheck() throws Exception {
+        final DefaultConfiguration checkConfig =
+                createModuleConfig(UnclosedTagIntolerantCheck.class);
+        UnclosedTagIntolerantCheck.violateJavadocWithNonTightHtml = true;
+        final String[] expected = {
+            "6:4: " + getCheckMessage(MSG_UNCLOSED_HTML, "P_TAG_START"),
+            "13:9: " + getCheckMessage(MSG_UNCLOSED_HTML, "P_TAG_START"),
+            "16:8: " + getCheckMessage(MSG_UNCLOSED_HTML, "LI_TAG_START"),
+            "22:16: " + getCheckMessage(MSG_UNCLOSED_HTML, "LI_TAG_START"),
+            "30:13: " + getCheckMessage(MSG_UNCLOSED_HTML, "LI_TAG_START"),
+            "39:22: " + getCheckMessage(MSG_UNCLOSED_HTML, "P_TAG_START"),
+        };
+        verify(checkConfig, getPath("InputUnclosedTag.java"), expected);
```

Comment:
Inputs should be named after their test.
It should start with `InputAbstractJavadoc`...